docs/source/glossary.rst

```diff
@@ -23,10 +28,31 @@ Glossary of Terms
     Event
         A well-defined occurrence at a specific point in :term:`time`.
         Events may occur
-        as result of activities ("dinner is done"),
+        as result of activities ("when dinner is done"),
         as time passes ("after 20 time units"),
         or
         at predefined points in time ("at 2000 time units"),
 
     Notification
         Information sent to an :term:`activity`, usually in response to an :term:`event`.
+        Notifications can only be received when an :term:`activity` is :term:`suspended <Suspension>`.
```

Comment:
Being postponed is a special case of being suspended. Should it be made clear that both apply?

Reply:
Ok, definition of Postponement and Suspension should maybe made more clear. You could put into the description of Postponement that this is a special case of being suspended. This might help.